python/GafferUI/GraphEditor.py

```diff
@@ -381,7 +382,7 @@ def __keyPress( self, widget, event ) :
 
 		return False
 
-	def __frame( self, nodes, extend = False ) :
+	def __frame( self, nodes, extend = False, at = None ) :
```

Comment:
This is a slightly weird API, since specifying both `extend` and `at` doesn't seem to make much sense, but for something internal, this isn't much of a concern.

Reply:
Yeah, `at` doesn't make much sense if `len( nodes ) > 1 )` either, because it can push some of them offscreen, but this is private and I didn't think that splitting into additional functions would necessarily improve clarity.